release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -121,6 +121,44 @@ io:println(nameText); // "DanBrown"
 
 ##### Language Server
 
+##### Ballerina Shell
+
+Ballerina Shell now supports redefining module-level declarations and variable declarations. `/remove NAMES` command can be used to remove one or multiple declarations from snippet memory.
```

Comment:
By module-level declarations I'm assuming you mean functions, types, etc.? Shall we say definitions instead?
```suggestion
- Ballerina Shell now supports redefining module-level definitions and variable declarations. The `/remove` command can be used to remove one or more declarations from snippet memory.
```

Also need to use bullet points, right?